provider/sidecar_test.go

```diff
@@ -92,7 +93,37 @@ func TestSidecar(t *testing.T) {
 					ID:       "008",
 					Name:     "api",
 					Hostname: "another-aws-host",
+					Status:   0,
+					Ports: []service.Port{
+						service.Port{
```

Comment:
`gofmt -s` on this file and it will probably remove these inner `service.Port` names.